src/lib/Fullscreen.js

```diff
@@ -1,4 +1,6 @@
 import EventEmitter from 'events';
+import fscreen from 'fscreen';
```

Comment:
I would change all functions below to also use it...so that we don't have a mix of native vs lib.

Reply:
@priyajeet I will create a ticket to refactor this file